frontend/language/src/nb.json

```diff
@@ -14,7 +14,7 @@
   "address_component.validation_error_house_number": "Bolignummer er ugyldig",
   "address_component.validation_error_zipcode": "Postnummer er ugyldig",
   "api_errors.AD_01": "Navnet er allerede i bruk for en annen sidegruppe.",
-  "api_errors.AD_02": "Navnet på sidegruppen kan ikke være tomt.",
+  "api_errors.AD_02": "Ugyldig navn på sidegruppe.",
```

Comment:
Can we add some more information about what a valid name is? 😊

Reply:
Jepp! Tror bare det kan være et lite issue knyttet til det vi snakket om rundt det endepunktet på refinement - dersom det skal returnere success dersom man prøver å legge til et layoutset som eksisterer så vil man ikke få noe api-error i dette tilfellet lenger. Men jeg tror egt ikke det gjør noe. Med den FE valideringen som er lagt til der nå så skal det ikke gjøres noe backend-kall dersom man fyller inn et ugyldig navn. Hva tenker du? 😊

Reply:
Brukeren vil nok ikke komme til denne feilmeldingen på grunn av FE-valideringen, som du nevner. Men kan denne meldingen bli brukt ved innlasting for å vise at noe er feil, dersom man har endret navn i kode? 🤔

Reply:
Den exceptionen blir foreløpig bare kastet fra backend fra PUT og POST kallet, så tror egt ikke det 🤔